docs/DeveloperGuide.md

```diff
@@ -320,6 +326,28 @@ Step 6. A `CommandResult` is produced based on whether the execution was a succe
 
 A similar sequence of events will occur when executing the `unmarkP` command, except that the `isCompleted` attribute of each `Project` will be set to `false` instead of `true`.
 
+### Add Task feature
+![AddTSequenceDiagram](images/AddTSequenceDiagram.png)
```

Comment:
The sequence diagram seems correct, but earlier the rest of the team decided not to include the relevant `CommandParser` classes to make the diagram simpler. I think it's up to you as to whether you still want to include it.

Reply:
I'll still include it for this command as I was working on the sequence diagram based on the `DeleteEmployeeCommand`, but thanks for the heads up!